frontend/src/pages/Home/index.tsx

```diff
@@ -25,6 +27,15 @@ const Home = () => {
           <CTA
             variant="contained"
             color="primary"
+            sx={{
```

Comment:
Don't put sx here, instead put it in styled CTA component.
https://github.com/bsoc-bitbyte/busify/blob/d15a1b516fae2e8f80adc16ef8a30eac7f333906/frontend/src/pages/Home/index.tsx#L122